dagmc_stats.py

```diff
@@ -42,12 +44,56 @@ def get_entity_ranges(my_core, meshset, entity_types):
 
     outputs
     -------
-    entity_ranges : a dictionary with one entry for each entity type that is a Range of handles to that type
+    native_ranges : a dictionary with one entry for each entity type that is a Range of handles to that type
     """
 
-    entity_ranges = {}
+    native_ranges = {}
     for entity_type in entity_types:
-        entity_ranges[entity_type] = my_core.get_entities_by_type(meshset, entity_type) 
+        native_ranges[entity_type] = my_core.get_entities_by_type(
+            meshset, entity_type)
+    return native_ranges
+
+
+def get_entityset_ranges(my_core, meshset, geom_dim):
+    """
+    Get a dictionary with MOAB Ranges that are specific to the types.MBENTITYSET type
+
+    inputs
+    ------
+    my_core : a MOAB Core instance
+    meshset : the root meshset for the file
+    geom_dim : the tag that specifically denotes the dimesion of the entity
 
-    return entity_ranges
+    outputs
+    -------
+    entityset_ranges : a dictionary with one entry for each entityset type, and the value is the range of entities that corrospond to 
+                        each type
+
+    """
+    entityset_ranges = {}
+    entityset_types = ['Nodes', 'Curves', 'Surfaces', 'Volumes']
+    for set_number, set_type in enumerate(entityset_types):
+        entityset_ranges[set_type] = my_core.get_entities_by_type_and_tag(meshset, types.MBENTITYSET, geom_dim,
+                                                                          [set_number])
+    return entityset_ranges
+
+
+def get_surfaces_per_volume(my_core, entityset_ranges):
+    """
+    Get the number of surfaces that each volume in a given file contains
+
+    inputs
+    ------
+    my_core : a MOAB core instance
+    entity_ranges : a dictionary of the entityset ranges of each tag in a file
+
+    outputs
+    -------
+    freqs : a list of the number of surfaces each volume in the file contains
+
+    """
 
+    freqs = []
```

Comment:
Also - this data is probably not actually frequencies, so it may be best to use a different name.  Accurate variable naming is good for improving long term maintainability.